ui/src/criteria/concept.tsx

```diff
@@ -44,6 +44,9 @@ type EntityConfig = {
   sourceConcepts?: boolean;
   attributes?: string[];
 
+  orderBy?: string;
+  orderDirection?: string;
```

Comment:
Make this a `tanagra.OrderByDirection` instead. Then it will automatically deserialize as the right type and you won't need the comparison below.